modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/osb-site-initializer-evp-test/src/testFunctional/tests/EVPOrganizationForm.testcase

```diff
@@ -115,10 +115,10 @@ definition {
 		}
 	}
 
-	@description = "LPS-162894 - Verify that after submit the form, when click on 'Done' button, will be redirected back to 'Organization' page"
+	@description = "LPS-162894 - Verify that after submit the form, when click on 'Done' button, will view a 'Thank you' message and be redirected back to 'Organization' page"
 	@priority = 2
 	test CanSeeRedirectionToOrganizationPageAfterCreateOrganization {
-		task ("When the employee user create an organization") {
+		task ("When the Employee user create an organization") {
```

Comment:
Just a typo here, could you change this task to: `"When the Employee user creates an organization"`